data/electronToChromium.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  "1.5": 54,
```

Comment:
I think it is okay to track this here. there is https://atom.io/download/electron/index.json which contains all the specific versions so this file could be generated from that.